src/redux/actions/workspaces-actions.ts

```diff
@@ -10,3 +15,30 @@ export const fetchWorkspace = (ws: string) => ({
   type: ActionTypes.FETCH_WORKSPACE,
   payload: WorkspacesHelper.getWorkspace(ws),
 });
+
+export const createWorkspace = (config: WorkspacesBasicWorkspace) => {
+  const cache = createIntlCache();
+  const intl = createIntl({ locale, messages: providerMessages as any }, cache); // eslint-disable-line @typescript-eslint/no-explicit-any
+
+  return {
+    type: ActionTypes.CREATE_WORKSPACE,
+    payload: WorkspacesHelper.createWorkspace(config),
+    meta: {
+      notifications: {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        rejected: (payload: any) => ({
```

Comment:
We can type the payload as object with detail
```suggestion
        rejected: (payload?: { detail: string }) => ({
```